packages/react-native-sdk/src/components/call/LobbyView.tsx

```diff
@@ -129,8 +130,8 @@ export const LobbyView = ({ joinButton }: LobbyViewType) => {
         <View style={styles.info}>
           <Text style={styles.infoText}>
             You are about to join a call with id {call?.id} at Stream.{' '}
-            {count
-              ? `${count}  more people are in the call now.`
+            {participantsCount !== 0
+              ? `${participantsCount} more people are in the call now.`
```

Comment:
Because 
```ts
> undefined !== 0
true
```
This gives You are about to join a call with id X at Stream. undefined more people...

Reply:
"1 more people..." sounds weird. Can we make it "X participant(s) is/are already in the call)

Reply:
Good catch..

shall we change it to `count >= 2`

Reply:
I didn't find undefined people. For that case, the `You are first to Join the call.` is rendered.